src/index.html

```diff
@@ -4,9 +4,474 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container container--center">
+      <header class="header">
+        <a
+          href="index.html"
+          class="header__logo"
+        >
+          <img
+            class="logo-img"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#Apple"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#Samsung"
+              >
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#Smartphones"
+              >
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#computers"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#Gadgets"
+              >
+                Gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#Tablets"
+              >
+                Tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#Photo"
+              >
+                Photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#Video"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <main class="catalog">
+      <div
+        class="catalog__card card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="product-img"
+        >
+
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock
+          (MNED2UA/A)
+        </div>
+
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
```

Comment:
use semantic tags for text elements. Check other cases
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>

        <p class="card__product-code">
          Product code: 195434
        </p>
```